<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameStudet</class>
 <widget class="QFrame" name="FrameStudet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitterH">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitterV">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="comboBoxDepartment"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="comboBoxYear"/>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxGroup">
            <item>
             <property name="text">
              <string>All</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>11</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>12</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>13</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>14</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>15</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>16</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>17</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>18</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>19</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>21</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>22</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>23</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>24</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>25</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>26</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>27</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>28</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>29</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>30</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>31</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>32</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>33</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>34</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>35</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>36</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>37</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelFindStudent">
          <property name="text">
           <string>Find Student</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditFind"/>
        </item>
        <item>
         <widget class="QTreeView" name="treeViewStudents">
          <property name="minimumSize">
           <size>
            <width>260</width>
            <height>400</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelTotalStudents">
          <property name="text">
           <string>Total Students: </string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTreeWidget" name="treeWidgetContract">
       <property name="minimumSize">
        <size>
         <width>260</width>
         <height>0</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBoxStudent">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>350</height>
          </size>
         </property>
         <property name="title">
          <string>Student Card</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <layout class="QVBoxLayout" name="verticalLayoutSurname">
              <item>
               <widget class="QLabel" name="labelSurname">
                <property name="text">
                 <string>Surname</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditSurname"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayoutName">
              <item>
               <widget class="QLabel" name="labelName">
                <property name="text">
                 <string>Name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditName"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayoutPatronym">
              <item>
               <widget class="QLabel" name="labelPatronym">
                <property name="text">
                 <string>Patronym</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditPatronym"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayoutSex">
              <item>
               <widget class="QLabel" name="labelSex">
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Sex</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBoxSex">
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="editable">
                 <bool>true</bool>
                </property>
                <property name="currentIndex">
                 <number>-1</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <layout class="QVBoxLayout" name="verticalLayoutPassportSeries">
              <item>
               <widget class="QLabel" name="labelPassportSeries">
                <property name="text">
                 <string>Passport Series</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBoxPassportSeries">
                <property name="maximum">
                 <number>9999</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayoutPassportNumber">
              <item>
               <widget class="QLabel" name="labelPassportNumber">
                <property name="text">
                 <string>Passport Number</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBoxPassportNumber">
                <property name="maximum">
                 <number>999999</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayoutPassportAuthority">
              <item>
               <widget class="QLabel" name="labelPassportAuthority">
                <property name="text">
                 <string>Passport Authority</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditPassportAuthority"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <layout class="QVBoxLayout" name="verticalLayoutRegistrationAdress">
              <item>
               <widget class="QLabel" name="labeRegistrationAdress">
                <property name="text">
                 <string>Registration Adress</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditRegistrationAdress"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayoutRegistrationDate">
              <item>
               <widget class="QLabel" name="labelRegistrationDate">
                <property name="text">
                 <string>Registration Date</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="dateEditRegistrationDate">
                <property name="displayFormat">
                 <string notr="true">dd.MM.yyyy (ddd)</string>
                </property>
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="5" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <layout class="QVBoxLayout" name="verticalLayoutPostcode">
              <item>
               <widget class="QLabel" name="labelPostcode">
                <property name="text">
                 <string>Postcode</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBoxPostcode">
                <property name="wrapping">
                 <bool>false</bool>
                </property>
                <property name="suffix">
                 <string notr="true"/>
                </property>
                <property name="prefix">
                 <string notr="true"/>
                </property>
                <property name="maximum">
                 <number>999999</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayoutPhoneV">
              <item>
               <widget class="QLabel" name="labelPhone">
                <property name="text">
                 <string>Phone Number</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayoutPhoneH">
                <item>
                 <widget class="QComboBox" name="comboBoxCountryCallingCode">
                  <property name="editable">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelPhoneBracketLeft">
                  <property name="text">
                   <string>(</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditPhoneCallingCode">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>40</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>40</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="inputMask">
                   <string>000; </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelPhoneBracketRight">
                  <property name="text">
                   <string>)</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditPhoneNumber">
                  <property name="inputMask">
                   <string notr="true">000-00-00; </string>
                  </property>
                  <property name="text">
                   <string notr="true">--</string>
                  </property>
                  <property name="cursorPosition">
                   <number>9</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayoutEmail">
              <item>
               <widget class="QLabel" name="labelEmail">
                <property name="text">
                 <string>Email</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditEmail"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayoutIndividualTaxpayerIdentificationNumber">
              <item>
               <widget class="QLabel" name="labelIndividualTaxpayerIdentificationNumber">
                <property name="text">
                 <string>Individual Taxpayer Identification Number</string>
                </property>
                <property name="buddy">
                 <cstring>labelIndividualTaxpayerIdentificationNumber</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditIndividualTaxpayerIdentificationNumber">
                <property name="inputMask">
                 <string notr="true">999999999999; </string>
                </property>
                <property name="maxLength">
                 <number>12</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="6" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>33</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <layout class="QVBoxLayout" name="verticalLayoutDateOfBirth">
              <item>
               <widget class="QLabel" name="labelDateOfBirth">
                <property name="text">
                 <string>Date Of Birth</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="dateEditDateOfBirth">
                <property name="currentSection">
                 <enum>QDateTimeEdit::DaySection</enum>
                </property>
                <property name="displayFormat">
                 <string notr="true">dd.MM.yyyy (ddd)</string>
                </property>
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayoutPlaceOfBirth">
              <item>
               <widget class="QLabel" name="labelPlaceOfBirth">
                <property name="text">
                 <string>Place Of Birth</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBoxPlaceOfBirth">
                <property name="editable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayoutCitizenship">
              <item>
               <widget class="QLabel" name="labelCitizenship">
                <property name="text">
                 <string>Citizenship</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBoxCitizenship">
                <property name="editable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="7" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="pushButtonAddStudent">
              <property name="text">
               <string>Add Student</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Del Student</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonApplyChanges">
              <property name="text">
               <string>Apply Changes</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <layout class="QVBoxLayout" name="verticalLayoutPassportDateOfIssue">
              <item>
               <widget class="QLabel" name="labelPassportDateOfIssue">
                <property name="text">
                 <string>Passport Date Of Issue</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="dateEditPassportDateOfIssue">
                <property name="currentSection">
                 <enum>QDateTimeEdit::DaySection</enum>
                </property>
                <property name="displayFormat">
                 <string notr="true">dd.MM.yyyy (ddd)</string>
                </property>
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayoutPassportSubdivisionCode">
              <item>
               <widget class="QLabel" name="labelPassportSubdivisionCode">
                <property name="text">
                 <string>Passport Subdivision Code</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBoxPassportSubdivisionCode">
                <property name="maximum">
                 <number>999999</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayoutRegistrationAdministrativeUnit">
              <item>
               <widget class="QLabel" name="labelRegistrationAdministrativeUnit">
                <property name="text">
                 <string>Registration Administrative Unit</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBoxRegistrationAdministrativeUnit">
                <property name="editable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QMdiArea" name="mdiArea">
         <property name="activationOrder">
          <enum>QMdiArea::CreationOrder</enum>
         </property>
         <property name="viewMode">
          <enum>QMdiArea::SubWindowView</enum>
         </property>
         <property name="documentMode">
          <bool>false</bool>
         </property>
         <property name="tabsClosable">
          <bool>false</bool>
         </property>
         <property name="tabsMovable">
          <bool>false</bool>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
